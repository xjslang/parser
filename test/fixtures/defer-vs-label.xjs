function testDeferVsLabel() {
  console.log('start')
  
  // This should be treated as defer statement
  defer console.log('defer cleanup')
  
  // This should be treated as regular label
  defer: {
    console.log('regular label block')
  }
  
  console.log('end')
}

testDeferVsLabel()
