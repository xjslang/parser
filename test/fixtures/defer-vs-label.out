start
regular label block
end
defer cleanup
